### ISSUE TITLE
Theme is an empty object within .attrs() methods

### ISSUE THREAD

**Issue by danhoran** [styled-components/styled-components#1261]
## Version
2.2.1

## Reproduction
https://www.webpackbin.com/bins/-KxA-1yjG-5iLGqyDdCB

## Steps to reproduce
See webpackbin above.

I've tested on 2.2.0 and everything is working fine.

## Expected Behavior
Methods defined within .attrs({}) should have a reference to the theme.

## Actual Behavior
Methods defined within .attrs({}) lose their reference to theme (theme is an empty object).

**Comment by mxstbr** [comment]
PRs to fix this appreciated!

**Comment by sqbiz** [comment]
Looks like this was fixed in 2.2.2, see:

https://www.webpackbin.com/bins/-KxRAd7Mi5bi53HS9jdl

Also, in the bug example, this line:

```
color: ({ theme }) => console.log(theme) && theme.color
```

Needs to be

```
color: ({ theme }) => { console.log(theme);  return theme.color }
```

Unless this is some next-stage ES syntax.

**Comment by mxstbr** [comment]
Yep, very true @sqbiz! Thanks!
